content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -798,53 +798,100 @@ updates:
 
 ### `versioning-strategy`
 
-When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, it uses the following overall strategies:
+When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, there are several different potential _versioning-strategies_:
 
-- For apps, the version requirements are increased, for example: npm, pip and Composer.
-- For libraries, the range of versions is widened, for example: Bundler and Cargo.
+| Option | Supported by | Action |
```

Comment:
Haha, great catch! I meant to delete it as part of the point of this PR is that users typically want to index by ecosystem => strategy, rather than the other way around. Fixed!